### DISCUSSION TITLE
Latest scraper doesn't scrape all following/subscriptions?

### DISCUSSION THREAD

**Discussion by richiema32** [UltimaHoarder/UltimaScraper#1101] [Q&A]
Authorises fine, but only sees ~13-15 models even though I've got quite a few more than that. Should you type in 16 (which correlate to any model) it gives a list error.

**Comment by UltimaHoarder** [comment]
Use latest commit

**Reply by RescueCat** [reply]
I'm also having this issue, I'm following subscribed to 50+ but only 12 show up. Using the latest build.
